libraries/provider_dnsimple_record.rb

```diff
@@ -79,9 +79,9 @@ def create_record
 
       def delete_record
         converge_by("delete record #{@new_resource.record_name} from domain #{@new_resource.domain}") do
-          dnsimple_client.zones.delete_record(dnsimple_client_account_id,
-                                              @current_resource.domain,
-                                              existing_record_id)
+          dnsimple_client.zones.delete_zone_record(dnsimple_client_account_id,
+                                                   @current_resource.domain,
+                                                   existing_record_id)
```

Comment:
I suggest parens on own line
```suggestion
                                                   existing_record_id
                                                   )
```